### DISCUSSION TITLE
NO $32 OPTION ?

### DISCUSSION THREAD

**Discussion by MICKYJUICE84** [arkypita/LaserGRBL#1576] [Q&A]
Hey guys im new to laser , just purchased a 3018 cnc with 5w laser, i have downloaded lasergrbl and connected to cnc all ok, has movement in all axis , x, y , z  but cannot get the laser to work , i have checked the config listing and there is no $30 or $32 in list 
![no$32](https://user-images.githubusercontent.com/90947017/133871349-61d36d9c-e1de-4807-9c65-cc6a8fa22c15.png)

is there something im missing here , i can use lightburn without issues , laser works perfect, just cannot get laser to fire on lasergrbl ?? please help a nooby lol 

cheers 
Mike

**Comment by arkypita** [comment]
Read [lasergrbl faq](https://lasergrbl.com/faq/) read "Help! Some buttons/options is missing" section.
